### DISCUSSION TITLE
[2135150]elin {艾琳}

### DISCUSSION THREAD

**Discussion by 1040302** [pjy612/SteamManifestCache#922] [Share]
### Steam 游戏ID

2135150

### 密钥列表

```hash
2135153;b800fa7ccfbea4d3c24ba4e8d50bdab1b39bf15b492ac44ad91b5af2af02cc14
```

### 清单附件

[2135150_16812418.zip](https://github.com/user-attachments/files/18218180/2135150_16812418.zip)

### 备注

1040302669@qq.com

**Comment by pjy612** [comment]
* 密钥格式错误
* #26

**Reply by 1040302** [reply]
打错符号了，这样应该是对的吧？
